lib/open_api/processor/type.ex

```diff
@@ -47,8 +52,11 @@ defmodule OpenAPI.Processor.Type do
   """
   @type primitive ::
           :boolean
+          | {:boolean, boolean_format}
           | :integer
+          | {:integer, integer_format}
           | :number
+          | {:number, number_format}
```

Comment:
Do we still need an atom-type or should we use `:generic` as a second tuple element as the "default" value